docs_src/microservices/core/metadata/Ch-Metadata.md

```diff
@@ -4,148 +4,435 @@
-|DeviceResource|	The atomic description of a particular protocol level interface for a class of Devices.	 |
-|DeviceProfile|	The description of both the protocol level interface, device service interface, and mapping and interpretation logic that describe communication to a class of devices. |	Top Level object
-|DeviceReport|	 	 |
-|DeviceService|	The current state and reachability information for a registered Device Services	 |
-|OperatingState|	An object’s current operating state of “Enabled” or “Disabled.”	 |
-|ProfileProperty|	The transformation, constraint, and unit properties for a class of Device data.	 |
-|ProfileResource|	The set of operations that is executed by a Service for a particular Command.	 |
-|PropertyValue|	The transformation and constraint properties for a class of data.	 |
-|ProvisionWatcher|	The metadata used by a Service for automatically provisioning matching Devices.	 |
-|ResourceOperation|	An Operation or set of Operations executed by the Device Service on a Device.	 |
-|Response|	A description of a possible REST response for a Command.	 |
-|Schedule|	An object defining a timer or alarm.|Top Level object|
-|ScheduleEvent|	The action taken by a Service when the schedule triggers.|Top Level object|
-|Service|	The current state and reachability information registered for a Service.	 |
-|Units|	The unit metadata about a class of Device data.	 |
+-   It is not responsible for actual data collection from
+    devices, which is performed by device services and core
+    data
+-   It is not responsible for issuing commands to the
+    devices, which is performed by core command and device
+    services
 
-## High Level Interaction Diagrams
+## Data Models
+
+To understand metadata, its important to understand the EdgeX data objects it manages.  Metadata stores its knowledge in a local persistence database.  [Redis](https://redis.io/) is used by default, but a database abstraction layer allows for other databases to be used.
+
+!!! Note
+        EdgeX has used [MongoDB](https://www.mongodb.com/) in the past. Through the Geneva release, MongoDB is still supported but is considered deprecated.  Alternate (both open source and commercial) implementations have also been provided in the past. 
+
+### Device Profile
+
+Device profiles define general characteristics about devices, the data they provide, and how to command them. Think of a device profile as a template of a type or classification of device. For example, a device profile for BACnet thermostats provides general characteristics for the types of data a BACnet thermostat sends, such as current temperature and humidity level. It also defines which types of commands or actions EdgeX can send to the BACnet thermostat.  Examples might include actions that set the cooling or heating point.  Device profiles are typically specified in YAML file and uploaded to EdgeX.  More details are provided below.
+
+#### Device Profile Details
```

Comment:
I am surprised how much information we have about devices and device profile in the metadata section. It seems to me this should provide a link over to devices services section and describe what goes into one over there. Yet, we're describing every piece of the device profile document in metadata. I think the relationship is necessary to describe here, but details about what makes up a profile doesn't seem appropriate here.